web/packages/teleport/src/Discover/ConnectMyComputer/TestConnection/TestConnection.tsx

```diff
@@ -94,36 +94,68 @@ export const TestConnection = (props: AgentStepProps) => {
           {meta.resourceName}
           &rdquo; by starting a session.
         </Text>
-        {refetchNodeAttempt.status === '' ||
-          (refetchNodeAttempt.status === 'processing' && <Indicator />)}
-
-        {refetchNodeAttempt.status === 'error' && (
-          <>
-            <TextIcon mt={2} mb={3}>
-              <Icons.Warning size="medium" ml={1} mr={2} color="error.main" />
-              Encountered Error: {refetchNodeAttempt.statusText}
-            </TextIcon>
-
-            <ButtonPrimary
-              type="button"
-              onClick={() => refetchNode(abortController.current.signal)}
+        {fetchLoginsAttempt.status === '' ||
+          (fetchLoginsAttempt.status === 'processing' && <Indicator />)}
+
+        {fetchLoginsAttempt.status === 'error' &&
+          (fetchLoginsAttempt.error instanceof ApiError &&
+          fetchLoginsAttempt.error.response.status === 404 ? (
+            <ErrorWithinStep
+              buttonText="Reload"
+              buttonOnClick={() => window.location.reload()}
```

Comment:
i think else where the language we use is `refresh` 🤷‍♀️

Reply:
Cool, I'll update it to reflect that.